src/constraints/scalaraffine.jl

```diff
@@ -196,18 +196,65 @@ function MOI.modify!(model::LinQuadOptimizer, index::LCI{S}, change::MOI.ScalarC
     change_matrix_coefficient!(model, row, column, change.new_coefficient)
 end
 
+function _replace_with_matching_sparsity!(model::LinQuadOptimizer, previous::Linear, replacement::Linear, row)
```

Comment:
I know the rest of the code doesn't, but can we add docstrings for these new functions to remind ourselves of why they are needed and what they do?

Reply:
done